installers/nix-setup-template.sh

```diff
@@ -38,7 +38,7 @@ rm $PYTHON_TOOLCACHE_VERSION_ARCH_PATH/setup.sh
 
 cd $PYTHON_TOOLCACHE_VERSION_ARCH_PATH
 
-echo "Create additional symlinks (Required for UsePythonVersion VSTS task)"
+echo "Create additional symlinks (Required for the UsePythonVersion ADO task and the setup-python GitHub Action)"
```

Comment:
```suggestion
echo "Create additional symlinks (Required for the UsePythonVersion Azure Pipelines task and the setup-python GitHub Action)"
```

("ADO" shouldn't be used as an abbreviation per branding guidelines)